apps/src/templates/projects/FeaturedProjectsTable.jsx

```diff
@@ -151,6 +151,17 @@ const feature = (channel, publishedAt) => {
     .fail(handleFeatureFailure);
 };
 
+const onDelete = channel => {
+  var url = `/featured_projects/${channel}`;
+  $.ajax({
```

Comment:
nit: can we use `fetch` here? looks like we have other usages of `$.ajax` in this file though so also okay if you want to keep it consistent (and we can upgrade this file up later as a follow-up)

Reply:
Sure! I'll wait till https://github.com/code-dot-org/code-dot-org/pull/56555 is merged since the authenticity token was required when `fetch` was used.